src/utility/Assert.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2019 Ole Kliemann, Malte Kliemann
+ *
+ * This file is part of DrMock.
+ *
+ * DrMock is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * DrMock is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with DrMock.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef DRMOCK_SRC_UTILITY_ASSERT_H
+#define DRMOCK_SRC_UTILITY_ASSERT_H
+
+#define DR_ASSERT(expr) do { \
+  if (not expr) \
```

Comment:
```suggestion
  if (not(expr)) \
```

Without extra parens, `DRUTILITY_ASSERT(4 == 5)` compiles on clang (albeit with a warning) but doesn't throw. This example should be part of a regression test.